src/main/resources/config/application.properties

```diff
@@ -37,17 +37,17 @@ core_case_data.jurisdictionId=${CORE_CASE_DATA_JURISDICTION_ID:SSCS}
 core_case_data.caseTypeId=${CORE_CASE_DATA_CASE_TYPE_ID:Benefit}
 
 # IDAM Java Client
-idam.url=${IDAM_API_URL:http://localhost:5000}
+idam.url=${IDAM_API_URL:http://localhost:5555}
```

Comment:
Is it okay to change to port 5555?
Some devs, (cough! me!) are using the real idam and not the stub.

Reply:
Good catch Ryan!  Mistaken checkin - thanks for spotting :-)